test/test.dart

```diff
@@ -257,7 +257,7 @@ Future<User> loginWithRetry(App app, Credentials credentials, {int retryCount =
     return await app.logIn(credentials);
   } catch (e) {
     if (retryCount > 1) {
-      await Future<User>.delayed(Duration(milliseconds: 150));
```

Comment:
This was like that in my PR https://github.com/realm/realm-dart/commit/fe3a5ef9ebe5387705dbdcd0c710616a438e2031: `await Future<dynamic>.delayed(Duration(milliseconds: 150));` 
I'm afraid there could be also other changes that were not merged.